crates/supervisor/storage/src/providers/log_provider.rs

```diff
@@ -84,7 +84,7 @@ where
                 incoming_block = %block,
                 "Incoming log block is not consistent with the stored log block",
             );
-            return Err(StorageError::ConflictError)
+            return Err(StorageError::ConflictError);
```

Comment:
```suggestion
            return Err(StorageError::ConflictError)
```
pls revert unrelated format changes like any new semicolons. it makes the diff unnecessarily large and makes the pr have scope creep. make sure to run the formatter with nightly toolchain only, ie `cargo +nightly fmt`